src/kit/Row.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { CSSProperties, ReactNode } from 'react';
+
+import css from './Row.module.scss';
+
+interface RowProps {
+  children?: ReactNode;
+  gap?: 0 | 8 | 16;
+  wrap?: boolean;
+  height?: number;
+  marginBottom?: 0 | 8 | 16;
```

Comment:
fix: marginBottom doesn't appear to be used anywhere or documented in the design kit. either it should be documented or removed

Reply:
realized we can just wrap a Row in a Section in the cases where that vertical spacing is needed, so will go ahead and remove.